controls/srg_ctr/SRG-APP-000092-CTR-000165.yml

```diff
@@ -3,4 +3,165 @@ controls:
   levels:
   - medium
   title: {{{ full_name }}} must initiate session auditing upon startup.
-  status: pending
+  rules:
+  - cluster_logging_operator_exists
+  - audit_log_forwarding_enables
```

Comment:
This rule doesn't contain an SRG reference to this control. Can you also check, while you are touching that rule, if the other two SRG references make sense for the rule?